opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -982,6 +986,7 @@ namespace Opm
     void
     MultisegmentWell<TypeTag>::
     getMobility(const Simulator& ebosSimulator,
+                const int seg,
```

Comment:
I think you could probably infer the segment from the perforation if you don't want to change the API.  On the other hand, that might just be an expensive way to recompute information you already have so I'm okay with adding the new parameter.

Reply:
Let us remove seg from the interface.